package.json

```diff
@@ -95,6 +95,7 @@
   },
   "resolutions": {
     "minimist": "1.2.6",
-    "postcss": "8.4.31"
+    "postcss": "8.4.31",
+    "braces": "^3.0.3"
```

Comment:
is there a better way to fix this? it's addressing this: https://www.npmjs.com/advisories/1097496